runtime/tests/checker/account_test.go

```diff
@@ -1083,6 +1083,33 @@ func TestCheckAccountContractsUpdate(t *testing.T) {
         `)
 		require.NoError(t, err)
 	})
+
```

Comment:
Can you please also add (a) test case(s) for `DeploymentResult` itself, i.e. access its field and "assert" the returned type matches?